### ISSUE TITLE
[Question][Hw4] About Infinite in interval domain

### ISSUE THREAD

**Issue by yongwoo36** [prosyslab-classroom/cs524-program-analysis#35]
Hello professor.
I have a question about interval domain in HW4.

While doing my homework, I came to think about the result of `add [MInf, MInf] [PInf, PInf]`.
I can't find the definition of the result of `MInf + PInf`.

But first, I wondered if an interval like `[MInf, MInf]` or `[PInf, PInf]` could exist, and I thought it couldn't. 
The reason is, for example, if my understanding is correct, the result of `[MInf, PInf] < [MInf, 10]` is `[MInf, 10]`, not `[MInf, MInf]`.

Therefore, as in the first example, there is no case where PInf comes on the left side of interval or MInf comes on the right side of the interval, so I thought that we do not have to consider these cases.

Am I correct? Or is there something I misunderstood?
If so, what is the result of `add [MInf, MInf] [PInf, PInf]`?

Thank you.

**Comment by hyunsukimsokcho** [comment]
MInf (or PInf) are not values to be rigorous, instead they are used to indicate any negative (or positive) value is included in the specified interval. Thus, both `[MInf, MInf]` and `[PInf, PInf]` are invalid interval. You really don’t need to consider them as operands when defining abstract operators.

**Comment by yongwoo36** [comment]
Thanks!

**Comment by ShinWonho** [comment]
Hello, I also have a question about infinite.

Can I assume Inf as finitely big number??
Why I ask this question is that `Inf * 0` is not defined.
In our homework, `mul [2, PInf] [0, 0]` can be the case.
Can I assume `mul [2, PInf] [0,0]` is [0,0]?

Thank you

**Comment by KihongHeo** [comment]
Yes. Here `Pinf` and `Minf` do not exactly mean the ones in (rigorous) math. They are just symbols representing "arbitrary large (or small)" numbers.

**Comment by ShinWonho** [comment]
Thank you professor!!
